nav2_collision_monitor/include/nav2_collision_monitor/collision_detector_node.hpp

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2022 Samsung R&D Institute Russia
+// Copyright (c) 2023 Pixel Robotics GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_COLLISION_MONITOR__DETECTOR_NODE_HPP_
+#define NAV2_COLLISION_MONITOR__DETECTOR_NODE_HPP_
```

Comment:
`NAV2_COLLISION_MONITOR__DETECTOR_NODE_HPP_` -> `NAV2_COLLISION_MONITOR__COLLISION_DETECTOR_NODE_HPP_` according to naming rules